### DISCUSSION TITLE
comment connaitre la précision de chaque image après le test

### DISCUSSION THREAD

**Discussion by SiwarRziguiING3** [ultralytics/yolov5#8083] [Q&A]
bonsoir, je veux connaitre où fait le calcule de confiance et où je peux trouvé la condition sur conf_thres dans le code.
merci de me rependre

**Comment by glenn-jocher** [comment]
@SiwarRziguiING3 confidence threshold is applied in NMS function here:
https://github.com/ultralytics/yolov5/blob/7cef03dddd6fba26fff6748ed1cfdd18208c193e/utils/general.py#L829-L836

**Reply by SiwarRziguiING3** [reply]
![images](https://user-images.githubusercontent.com/92124798/171859495-ccd59539-3708-4e1c-909e-5a9871a5fb08.jpg)
bonjour, je veux savoir où (' dans le code' ) je peux modifier les étiquettes des images après le test je veux juste afficher le nom de la classe détecté sans afficher la précision.
merci de me répondre

**Reply by glenn-jocher** [reply]
@SiwarRziguiING3 https://github.com/ultralytics/yolov5/blob/47233e1698b89fc437a4fb9463c815e9171be955/detect.py#L236

**Reply by SiwarRziguiING3** [reply]
je veux connaitre est-ce que  je peux remplacé  l'affichage de la précision  dans l'image après par une label  par exemple par 'non conforme' si oui où je peux modifier dans le code

**Reply by glenn-jocher** [reply]
@SiwarRziguiING3 you can change `label` here:
https://github.com/ultralytics/yolov5/blob/47233e1698b89fc437a4fb9463c815e9171be955/detect.py#L166-L169
